routes/userSearch.js

```diff
@@ -0,0 +1,28 @@
+var express = require("express");
+var router = express();
+var database = require("../database");
+
+router.get("/usersearch", function (request, response) {
+  let username = request.query.username;
+
+  if (username) {
+    database.query(
```

Comment:
this query will exactly match username, this query should return approximately similar name users.

for e.g if user types ama then it can return man, amandeep, amanta etc.